### PR TITLE
Cloud migrations: Add cloud migration to auditing docs

### PR DESCRIPTION
**Pull request by dana-axinte**
**What is this feature?**

Add cloud migration management to auditing docs.

image

**Why do we need this feature?**

Monitor activities around cloud migrations.

**Who is this feature for?**
Available in Grafana Enterprise for users using the Cloud Migration Assistant.

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [X] It works as expected from a user's perspective.
- [X] If this is a pre-GA feature, it is behind a feature toggle.
- [x] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

**Review by irenerl24 — APPROVED** [review]

File: docs/sources/setup-grafana/configure-security/audit-grafana.md [hunk 1]
```diff
@@ -326,6 +326,17 @@ external group.
 | Set licensing token      | `{"action": "create", "requestUri": "/api/licensing/token"}` |
 | Save billing information | `{"action": "billing-information"}`                          |
 
+#### Cloud migration management\*
```
Comment (knylander-grafana) [inline docs/sources/setup-grafana/configure-security/audit-grafana.md:329]: why not just put a statement here that this feature requires a feature toggle, list the feature toggle, and link to the feature toggle page. 

Is this feature experimental, private or public preview? If so, you can use one of the standard admonitions in the shortcodes. https://grafana.com/docs/writers-toolkit/write/shortcodes/#docsprivate-preview

Comment (knylander-grafana) [reply docs/sources/setup-grafana/configure-security/audit-grafana.md:329]: What about this? Do you want to also list the feature toggle?
```suggestion
#### Cloud migration management
 
Cloud migration management is available in Grafana Enterprise for users using the Cloud Migration Assistant.
You must activate a feature toggle to access this feature. 
  
```

Comment (dana-axinte) [reply docs/sources/setup-grafana/configure-security/audit-grafana.md]: image

Thanks so much for looking into this 🙏  I tried your suggestion and this is how it looks. I think just the public preview annotation is sufficient, but can leave both the text and the note. What do you think?

Comment (knylander-grafana) [reply docs/sources/setup-grafana/configure-security/audit-grafana.md]: This looks great. The public preview annotation is enough. I had written the example before putting in the admonition.

**Review by knylander-grafana — APPROVED** [review]
This looks great! Thank you for updating the documentation.

**Comment by knylander-grafana** [comment]
I commited a small change (single space) to trigger the build checks. The PR checks appear to have gotten stuck.

**Comment by knylander-grafana** [comment]
The drone build is failing on the prettier check. You can fix this by running "prettier -w /path/to/doc/file" and then commiting the outputted file.

**Pull request merged**